Client/Frontend/Intro/IntroViewController.swift

```diff
@@ -364,6 +349,11 @@ class IntroViewController: UIViewController, UIScrollViewDelegate {
         label.numberOfLines = 0
         label.attributedText = attributedStringForLabel(text)
         label.font = IntroViewControllerUX.CardTextFont
+
+        addLabelsToIntroView(introView, label: label, title: title)
+    }
+
+    private func addLabelsToIntroView(introView: UIView, label: UIView, title: String = "") {
```

Comment:
nit: I don't like naming something that might not be a `UILabel` (i.e. it could be a `UIButton` as with the sync card) `label`. Maybe `detailView` or something?

Also, I think the function itself should be `addViewsToIntroView` or equivalent as we are not necessarily adding just labels.